core/cli/commands/codebase.py

```
@@ -59,6 +59,216 @@ def _git_python_files(repo_root: str, exclude: set[str] | None = None) -> list[s
+                if stale_ids:
+                    client.query(
+                        "FOR e IN @@edges FILTER e.type IN @types AND (e._from IN @ids OR e._to IN @ids) REMOVE e IN @@edges",
+                        bind_vars={"@edges": cols.edges, "types": rust_edge_types, "ids": stale_ids},
+                    )
+
+                # Now remove old symbols
+                client.query(
+                    "FOR s IN @@symbols FILTER s.file_path IN @paths REMOVE s IN @@symbols",
+                    bind_vars={"@symbols": cols.symbols, "paths": all_attempted_paths},
+                )
+            except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)